SPECS/glibc/glibc.spec

```diff
@@ -36,6 +36,7 @@ BuildRequires:  gettext
 BuildRequires:  kernel-headers
 BuildRequires:  texinfo
 Requires:       filesystem
+Requires:       pkgconfig(libcrypt)
```

Comment:
I think for this specific situation, we probably don't need to use pkgconfig? Just Requires: libcrypt should be enough I think? That will require the 'virtual' libcrypt package that only glibc-libcrypt and libxcrypt provide (via the Provides: we're adding to them).

Also it looks like we can set a preference: https://docs.fedoraproject.org/en-US/packaging-guidelines/WeakDependencies/#_package_preference

So probably we want to also include here:
Suggests: glibc-libcrypt